.actions/helpers.py

```diff
@@ -344,7 +348,21 @@ def _get_card_item_cell(path_ipynb: str, path_meta: str, path_thumb: Optional[st
         if dirname != ".notebooks":
             meta["tags"].append(dirname)
 
-        meta["tags"] = ",".join(meta["tags"])
+        tags = []
+        invalid_tags = []
+        for tag in meta["tags"]:
+            if " " in tag:
+                invalid_tags.append(tag)
+                continue
+            tags.append(HelperCLI.TAGS.get(tag, tag))
+
+        if invalid_tags:
+            raise ValueError(
+                f"The following tags are invalid: {','.join(invalid_tags)}. Tags should not contain spaces, use a "
```

Comment:
Why not, is it some docs issue?
If so why just not replace all spaces by -

Reply:
Yeah, it only supports dashes. We could just replace spaces, I'll change it to that

Reply:
Done :smiley: